docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -20,8 +20,8 @@ Now that we've reached [MUI X v6.11.0](https://github.com/mui/mui-x/releases/ta
   - [Support for time zones](#support-for-time-zones)
   - [Digital clock](#digital-clock)
 - [Data Grid](#data-grid)
-  - [Filter on column headers](#filter-on-column-headers) <a title="Pro plan" href="/x/introduction/licensing/#pro-plan"><span class="plan-pro"></span></a>
-  - [Copy and paste](#copy-and-paste) <a title="Premium plan" href="/x/introduction/licensing/#premium-plan"><span class="plan-premium"></span></a>
+  - [Filter on column headers](#filter-on-column-headers) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
```

Comment:
Maybe we could go with Title Case because it is used in the actual page that we link to? 🤔 
```suggestion
  - [Filter on column headers](#filter-on-column-headers) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro Plan')
```